.drone.yml

```diff
@@ -0,0 +1,48 @@
+workspace:
+  base: /srv/app
+  path: src/github.com/go-gitea/gitea
+
+pipeline:
+  testing:
+    image: webhippie/golang:edge
+    pull: true
+    environment:
+      CGO_ENABLED: 0
+    commands:
+      - make clean
+      - make vet
+#      - make lint fails currently with 500 errors
+      - make test
+      - make build
+
+  updater:
+    image: karalabe/xgo-latest:latest
+    pull: true
+    commands:
+      - make publish
+    when:
+      event: [ push, tag ]
+      branch: [ master, refs/tags/* ]
+
+  coverage:
+    image: plugins/coverage
+    server: http://coverage.gitea.io
```

Comment:
Because aircover is another nearly oss coverage tool we can also provide.

Just saying that because there is already a native plugin for that available.

We can also add gitea integration to that as well.